src/TestCase.php

```diff
@@ -115,6 +115,10 @@ protected function setUpTraits()
     {
         $uses = array_flip(class_uses_recursive(static::class));
 
+        if (isset($uses[RefreshDatabase::class])) {
```

Comment:
Don't we need to import this class to work?

Reply:
Yes sorry totally forgot, normally my IDE would import the classes. I have seen you added the missing import, thanks!